services/scrutinizer/scrutinizer.service.js

```diff
@@ -14,16 +14,25 @@ class ScrutinizerBuild extends LegacyService {
 
   static get route() {
     return {
-      base: 'scrutinizer',
-      pattern: '',
+      base: 'scrutinizer/build',
+      pattern: ':vcsType/:user/:repo',
     }
   }
 
   static get examples() {
     return [
       {
-        title: 'Scrutinizer Build',
-        previewUrl: 'build/g/filp/whoops',
+        title: 'Scrutinizer build',
+        namedParams: {
+          vcsType: 'g',
```

Comment:
No changes needed, but just an observation that since the first parameter has the subtext prefix `e.g. ` in the modal window, this one kinda looks like `egg` at first glance!

Reply:
Hehe, fortunately not something that comes up too much.